[FW][IMP] product_ux: dispatch report in format 10x15

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/product#516

jcadhoc committed:
[IMP] product_ux: dispatch report in format 10x15

X-original-commit: fbf5560d812cd73c436a57bdaf6052d83328b85e

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/product/pull/520).

fw-bot-adhoc commented:
@jcadhoc @vib-adhoc cherrypicking of pull request ingadhoc/product#516 failed.

stdout:
```
CONFLICT (modify/delete): product_ux/report/picking_templates.xml deleted in HEAD and modified in 9fdd71bc ([IMP] product_ux: dispatch report in format 10x15). Version 9fdd71bc ([IMP] product_ux: dispatch report in format 10x15) of product_ux/report/picking_templates.xml left in tree.
CONFLICT (modify/delete): product_ux/report/ir.action.reports.xml deleted in HEAD and modified in 9fdd71bc ([IMP] product_ux: dispatch report in format 10x15). Version 9fdd71bc ([IMP] product_ux: dispatch report in format 10x15) of product_ux/report/ir.action.reports.xml left in tree.
Auto-merging product_ux/__manifest__.py
CONFLICT (content): Merge conflict in product_ux/__manifest__.py

```

stderr:
```
13:57:43.352266 git.c:444               trace: built-in: git cherry-pick 9fdd71bc9b7b5f1399320782651071f379030dc0
error: could not apply 9fdd71bc... [IMP] product_ux: dispatch report in format 10x15
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

jcadhoc commented:
Cerramos ya que quedó viejo, este codigo ya se encuentra en v16 en stock_ux
